kong/include/kong/services/event_hooks/v1/event_hooks.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
```

Comment:
This file was never supposed to be part of OSS. Please remove it.

Reply:
done